emu_mps/mps_config.py

```diff
@@ -84,6 +85,7 @@ def __init__(
             max_krylov_dim=max_krylov_dim,
             extra_krylov_tolerance=extra_krylov_tolerance,
             num_gpus_to_use=num_gpus_to_use,
+            optimize_qubit_ordering=optimize_qubit_ordering,
```

Comment:
is that also an idea to let users specify their own qubit ordering?
If someone simulates same sequence multiple times with e.g. different dt, they shouldn't have to run optimatrix more than once.
If someone wants to use a different solver (Cplex), that would also be possible

Reply:
User can specify their order initially when preparing an interaction matrix according to their needs. So no use of the optimatrix is required. 

I would say that this trick with permuting indices is new, and not known in the community. I used Cplex and it takes several minutes to permute the matrix, while optimatrix does basic heuristics quickly around a second. This PR is a first order improvement, - adding a new optimisation solver can be considered as a next step